pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -77,6 +79,39 @@ def _gen_distributor():
     }
 
 
+def _sync_and_publish_repo(server_config, importer=None, distributors=None):
```

Comment:
Removal is fine; trimming down the function is fine; some other more ingenious solution is fine. So long as the solution fits the problem.